src/azure-cli/install.sh

```diff
@@ -197,6 +198,28 @@ if [ ${#AZ_EXTENSIONS[@]} -gt 0 ]; then
     done
 fi
 
+if [ "${AZ_INSTALLBICEP}" = "true" ]; then
+    # Properly install Azure Bicep based on current architecture
+    # The `az bicep install` command installs the linux-x64 binary even on arm64 devcontainers
+    # The `az bicep install --target-platform` could be a solution; however, linux-arm64 is not an allowed value for this argument yet
+    # Manually installing Bicep and moving to the appropriate directory where az expects it to be
+    apt-get install curl -y
```

Comment:
We might need an `apt update` in certain cases if the lists haven't been populated yet (it's not safe to assume it has, and each of our features `rm`'d the list at the end to reduce the size of each Feature layer)

Feel free to rip off this function used in a lot of other scripts in this repo (hopefully will move this to a centralized spot in the future, ref: https://github.com/devcontainers/spec/issues/129#issuecomment-1316040043)

https://github.com/devcontainers/features/blob/c4648ab98cdeeb520e7e629c5db4a471ec802954/src/git/install.sh#L89-L94

Reply:
Great call! I just noticed this script already includes the `check_packages` function so I will call that to ensure apt update is run and install curl.